bepolia/config.yaml

```
@@ -29,8 +29,8 @@ TERMINAL_BLOCK_HASH: 0x000000000000000000000000000000000000000000000000000000000
 TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH: 18446744073709551615
 
 # Capella
-CAPELLA_FORK_VERSION: 0x03001020
-CAPELLA_FORK_EPOCH: 18446744073709551615
+CAPELLA_FORK_VERSION: 0x00000072
```

Shouldn't it just be `BELLATRIX_FORK_VERSION(0x90000071)` + 1 

```suggestion
CAPELLA_FORK_VERSION: 0x90000072
```

---

yes!

@timbeiko we generally prefix fork versions with a non 0x00 prefix to avoid accidental signature/etc collisions with the 0x00 prefixed mainnet

---

That said, it's funny Sepolia started at 0x69.... a subtle troll by someone

---

Ah, got it - I went off what was used here: https://github.com/ethpandaops/withdrawals-testnet/blob/master/zhejiang-testnet/custom_config_data/config.yaml#L33